MediaRecorder.html

```diff
@@ -442,6 +443,20 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            Provides a list of the <a href="https://tools.ietf.org/html/rfc2046">
+            MIME types</a> supported by the platform. The returned list is ordered
+            so that MIME types closer to the beginning have a larger probability
+            of being supported by encode accelerators.
+            <div>
+              <em>Return type:</em> <code>sequence&lt;DOMString&gt;</code>
```

Comment:
It returns a promise, not a sequence directly. Wording more in the style of https://notifications.spec.whatwg.org/#dom-notification-requestpermission would be nice :)